### PR TITLE
Setting prefix explicitly in excelparser

### PR DESCRIPTION
**Pull request by francescalb**
# Description:
Prefix can now be set explicitly for imported ontologies in excelparser. Either only prefix of given ontology, or set common prefix for all ontologies with a given root in the base_iri. Importing subontologies is also corrected.

The following is an image of the ImportedOntologies excelsheet. 
image

The following image shows two new tests added in the excelparser test, in which two new classes are added. Both of these are subclasses on the locally imported testontology, either from the first import or its subontology. Note that they are given the same prefix based on the base_iri_root. If base_iri_root is not given the given prefix is only set to the main ontology and not the indirect imports. If base_iri_root is given all ontologies with their base_iri starting with base_iri_root are given the same prefix.

image

## Type of change:

- [ ] Bug fix.
- [x] New feature.
- [ ] Documentation update.

## Checklist:

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: ontopy/excelparser.py [hunk 1]
```diff
@@ -99,18 +100,19 @@ def create_ontology_from_excel(  # pylint: disable=too-many-arguments
...
         )
     except ValueError:
         pass
     else:
-        # Strip leading and trailing white spaces in path
-        imports.extend(
-            imports_frame["Imported ontologies"].str.strip().to_list()
-        )
+        # Strip leading and trailing white spaces in paths
+        imports = imports.replace(" ", "")
```
Comment (jesper-friis) [inline ontopy/excelparser.py:113]: This removes all spaces in path. Use `imports.strip()` to strip leading and trailing white spaces.

Comment (francescalb) [reply ontopy/excelparser.py:113]: Good point. I did not consider the possibility of adding the path to a local file with spaces in the path.

**Review by jesper-friis — APPROVED** [review]
Looks very nice

**Comment by CasperWA** [comment]
@francescalb I think this can be merged, and then you can try to perform a release again using all the new bells and whistles :)

**Pull request merged**